### PR TITLE
Update splash screen and about dialog

### PR DESCRIPTION
**Pull request by k-dominik**
Updated the splash screen to this one:

splash

font-wise I used the one we have on our webpage (istok-web) which I find more appealing than this bertha druq...

I have also added a dedicated about dialog:

about

### CODE REVIEW

File: ilastik/shell/gui/aboutDialog.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+        main_layout.addLayout(content_layout)
+        main_layout.addWidget(btn_box)
+
+        self.setLayout(main_layout)
+        self.setStyleSheet("background-color: white;")
+        self.setFixedSize(splash_pixmap.width() * 2.5, splash_pixmap.height() * 1.05)
+
+
+def showAboutDialog(parent=None):
```
Comment (emilmelnikov) [inline ilastik/shell/gui/aboutDialog.py]: Maybe make this a classmethod of AboutDialog?

File: ilastik/shell/gui/aboutDialog.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+)
+from PyQt5.QtCore import Qt
+from functools import partial
+
+
+import ilastik
+
+
+about_text = f"""
+    ilastik version {ilastik.__version__} <br /><br />
```
Comment (emilmelnikov) [inline ilastik/shell/gui/aboutDialog.py]: A very minor point, but one could use `<p>` instead of `<br>` (not sure if that's relevant outside the Web).

**Comment by k-dominik** [comment]
Hey @emilmelnikov, thank you very much for taking the time to look into this! Your input is highly appreciated and I followed both of your suggestions.  The cluncky name for the `showAboutDialog` still stays, since shorter alternatives like `show`, or `create` are already taken.

**Comment by akreshuk** [comment]
The new font is a real pleasure to look at after the old one. Do you use the same one for the main version and the subversion below? The 3s look a bit different. Otherwise, great!

**Comment by k-dominik** [comment]
> The new font is a real pleasure to look at after the old one. Do you use the same one for the main version and the subversion below? The 3s look a bit different. Otherwise, great!

Great eyes :). It's a non-system font. It is rendered to the image in inkscape. The subversion string is generated dynamically. We could ship the custom font with ilastik if we wanted to though (it's available with the open font license.

**Comment by emilmelnikov** [comment]
> The cluncky name for the `showAboutDialog` still stays, since shorter alternatives like `show`, or `create` are already taken.

Maybe `createAndShowModal` or something similar would be better? Now it's a bit little repetitive...

**Comment by k-dominik** [comment]
thx, that was a good one @emilmelnikov . changed it accordingly

**Pull request merged**